bundles/org.eclipse.jface/src/org/eclipse/jface/wizard/WizardDialog.java

```diff
@@ -1566,4 +1568,11 @@ public void run() {
 			});
 		}
 	}
+
+	@Override
+	protected IDialogSettings getDialogBoundsSettings() {
+		String name = getClass().getSimpleName() + ".dialogBounds"; //$NON-NLS-1$
+		IDialogSettings dialogSettings = getWizard().getDialogSettings();
```

Comment:
dialogSettings might be null

Reply:
Oh damn yes. Saw all the failing tests now. Fixed with next push